### PR TITLE
Feature/iesp 161

### PR DESCRIPTION
**Pull request by acolella92**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -90,6 +90,11 @@
 			<artifactId>validation-api</artifactId>
 			<version>${javax.validation.api}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>3.12.0</version>
```
Comment (nttdata-rtorsoli) [inline pom.xml:96]: Put version on the properties

File: src/main/java/it/pagopa/interop/probing/scheduler/dto/impl/EserviceContent.java [hunk 2]
```diff
@@ -11,7 +12,7 @@
 
 @Data
 @Builder
-public class EserviceContent {
+public class EserviceContent implements Dto {
```
Comment (nttdata-rtorsoli) [inline src/main/java/it/pagopa/interop/probing/scheduler/dto/impl/EserviceContent.java:15]: Named class with postfix Dto

File: src/main/java/it/pagopa/interop/probing/scheduler/dto/impl/PollingEserviceResponse.java [hunk 3]
```diff
@@ -9,7 +11,7 @@
 
 @Data
 @Builder
-public class PollingEserviceResponse {
+public class PollingEserviceResponse implements Dto {
```
Comment (nttdata-rtorsoli) [inline src/main/java/it/pagopa/interop/probing/scheduler/dto/impl/PollingEserviceResponse.java:14]: Response implements Response interface not dto

File: src/main/java/it/pagopa/interop/probing/scheduler/scheduler/ScheduledTasks.java [hunk 4]
```diff
@@ -39,24 +38,30 @@ public void scheduleFixedDelayTask() {
...
-      for (EserviceContent service : response.getContent()) {
-        try {
-          servicesSend.sendMessage(service);
-          eserviceService.updateLastRequest(service.getEserviceRecordId(),
-              ChangeLastRequest.builder().lastRequest(OffsetDateTime.now(ZoneOffset.UTC)).build());
-        } catch (IOException e) {
-          logger.logQueueSendError(service.getEserviceRecordId());
+    try {
+      Integer offset = 0;
+      while (true) {
```
Comment (nttdata-rtorsoli) [inline src/main/java/it/pagopa/interop/probing/scheduler/scheduler/ScheduledTasks.java:43]: Use Async development with Threads

**Review by nttdata-rtorsoli — CHANGES_REQUESTED** [review]
See comments

File: src/main/java/it/pagopa/interop/probing/scheduler/dto/impl/EserviceContent.java (hunk 2 above)
Comment (acolella92) [reply src/main/java/it/pagopa/interop/probing/scheduler/dto/impl/EserviceContent.java:15]: Actually this is not a dto, it is an internal data structure of a Response object, should i leave it without interface?

File: src/main/java/it/pagopa/interop/probing/scheduler/scheduler/ScheduledTasks.java (hunk 4 above)
Comment (acolella92) [reply src/main/java/it/pagopa/interop/probing/scheduler/scheduler/ScheduledTasks.java:43]: we already have IESP-140 task for this in our backlog

File: src/main/java/it/pagopa/interop/probing/scheduler/dto/impl/EserviceContent.java (hunk 2 above)
Comment (nttdata-rtorsoli) [reply src/main/java/it/pagopa/interop/probing/scheduler/dto/impl/EserviceContent.java:15]: Then it should have a Response interface

**Review by nttdata-rtorsoli — DISMISSED** [review]

File: src/main/java/it/pagopa/interop/probing/scheduler/dto/impl/PollingEserviceResponse.java (hunk 3 above)
Comment (acolella92) [reply src/main/java/it/pagopa/interop/probing/scheduler/dto/impl/PollingEserviceResponse.java]: done and pushed

File: pom.xml (hunk 1 above)
Comment (acolella92) [reply pom.xml]: done and pushed

**Review by galales — APPROVED** [review]

**Pull request merged**